Server/DeviceManagement/DeviceManagement.Domain/DeviceManagement.Domain.csproj

```diff
@@ -8,11 +8,13 @@
     </PropertyGroup>
 
     <ItemGroup>
+      <PackageReference Include="Microsoft.EntityFrameworkCore" Version="6.0.0" />
```

Comment:
Domain project must not know about DataAccess implementation details. Remove this reference.